### ISSUE TITLE
Undefined symbol: _ZN5boost6system15system_categoryEv error while trying to execute LM training

### ISSUE THREAD

**Issue by Henzi** [wangyu09/exkaldi#20]
Hello,

I am a student currently trying to use ExKaldi for my Master thesis as I think it would be perfect for me to use Kaldi with python without having to understand C++ in depth.
While trying to execute the tutorials to be able to reuse code and adapt it to mine, i got a few errors due to the shared libraries. I am using Ubuntu 20.04 and jupyter notebook (anaconda3). 
I downloaded the libraries that were missing on my computer (libboost_program_options, libboost_system and libboost_thread). However, I downloaded a version that is quite recent (version 1.71.0) and thus, I had to use symbolic links so that each time the code (exkaldisrc/tools/lmplz) tried to access the libraries in 1.65.1 version, it got access to the version available in my computer.
Then, while trying to train the kenlm N-Grams language model, i get the following error:
```
KenlmProcessError: Failed to generate arpa file.
.../anaconda3/exkaldisrc/tools/lmplz: symbol lookup error: .../anaconda3/exkaldisrc/tools/lmplz: undefined symbol: _ZN5boost6system15system_categoryEv.
```
After searchig on the internet I found out on StackOverflow that this error means that libboost_system is missing from the command line, and thus that i should add -lboost_system. Link: https://stackoverflow.com/questions/38650981/undefined-reference-to-symbol-zn5boost6system15system-categoryev-error

However, as I am using python on Jupyter Notebook and I am not familiar with C++, I do not understand exactly how to do that.
I think maybe it has to do something with the CMakeLists.txt that is present in the exkaldi-master/src but i am not familiar with CMake either. Do I have to add or modify some library flags in order to avoid this error from persisting?
Does this error has something to do with me downloading a different boost library version? 

I have been trying to solve this error for a few days without success. I hope someone can help me with it.
Thank you in advance

**Comment by luvwinnie** [comment]
Please install on the ubuntu with these.

```
sudo apt-get install libboost-all-dev
```

**Comment by Henzi** [comment]
Thank you for your reply. However, I already tried that command to install the libraries. In Ubuntu 20.04 the resulting libraries installed are in version 1.71.0. I found no command that was able to install the correct version.
I solved this issue by installing Ubuntu 18.04 and then using the command that you suggested!
